res/app/spotify-qt.desktop

```diff
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Type=Application
-Name=spotify-qt
+Name=Spotify-qt
```

Comment:
Any reason for changing this? The project is spelled in all lower-case, so the name in the desktop entry should as well.

Reply:
This change is just to make the app launcher look better in app menus and match the rest of applications.

Reply:
While I do somewhat agree, please keep this file as it is.

Reply:
Alright, its nothing big but makes name of app in menu entry look odd(with first character written in lowercase). I will revert it back.

Reply:
Reverted back.